proto/umee/leverage/v1/query.proto

```diff
@@ -260,14 +266,38 @@ message QueryBadDebtsResponse {
 // QueryMaxWithdraw defines the request structure for the MaxWithdraw gRPC service handler.
 message QueryMaxWithdraw {
   string address = 1;
-  // denom is the base token denom associated with the uToken to withdraw
-  string denom = 2;
+  // denom is the base token denom associated with the uToken to withdraw.
+  // empty denom will query all registered tokens.
+  optional string denom = 2;
```

Comment:
I've just checked in the documentation and SDK examples. 
* in SDK the `optional` flag is not used
* in the [protobuf docs](https://developers.google.com/protocol-buffers/docs/proto#optional) we read: "If the default value is not specified for an optional element, a type-specific default value is used instead: for strings, the default value is the empty string. For bytes, the default value is the empty byte string. "

For that reason, it doesn't make sense to put this optional - gogoproto makes the implementation a bit more obscure. Let's just use the empty string as it is defined in the comment.

Sorry for confusion.

Reply:
Got it